pac4j-jwt/src/main/java/org/pac4j/jwt/profile/JwtProfile.java

```diff
@@ -0,0 +1,13 @@
+package org.pac4j.jwt.profile;
+
+import org.pac4j.core.profile.CommonProfile;
+
+/**
+ * Represents a user profile based on a JWT.
+ *
+ * @author Misagh Moayyed
+ * @since 1.8.2
+ */
+public class JwtProfile extends CommonProfile {
```

Comment:
I'm wondering if we should inherit from `HttpProfile`, this is the case for all profiles built from data received in direct clients (LDAP, Mongo...)